src/database/migrations/20220411131632-create-trip-accomodation-relationship.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable valid-jsdoc */
+/* eslint-disable require-jsdoc */
+  export async function up(queryInterface, Sequelize) {
+    await queryInterface.addColumn(
+      'trips', 
+      'accomodationId', 
+      {
+      
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'accomodations', 
+          key: 'id', 
+        },
+        onUpdate: 'CASCADE',
+        onDelete: 'SET NULL',
```

Comment:
This should also be cascade. If we delete a user their trip info is not as helpful since it's owner is not there anymore.